examples/reliable_conversation/test_basic.py

```
@@ -0,0 +1,340 @@
+        anthropic=None
+    )
+    
+    app = MCPApp(name="rcm_fallback_test", settings=settings)
+    
+    @app.workflow
+    class FallbackTestWorkflow(ConversationWorkflow):
+        """Fallback test workflow"""
+        pass
+
+    try:
+        async with app.run() as test_app:
```

Ruff (0.11.9)

Local variable `test_app` is assigned to but never used

Remove assignment to unused variable `test_app`

(F841)